WalletWasabi.Fluent/ViewModels/Wallets/Home/History/CoinJoinDetailsViewModel.cs

```diff
@@ -20,7 +20,7 @@ public partial class CoinJoinDetailsViewModel : RoutableViewModel
 	[AutoNotify] private string _date = "";
 	[AutoNotify] private string _status = "";
 	[AutoNotify] private Money? _coinJoinFee;
-	[AutoNotify] private string _coinJoinFeeString;
+	[AutoNotify] private string _coinJoinFeeString = string.Empty;
```

Comment:
The convention is to use `""` instead of `string.Empty`

https://github.com/zkSNACKs/WalletWasabi/blob/e33531cc3da4d2d06a445e60ed1e2a979bc3028c/CONTRIBUTING.md

![image](https://user-images.githubusercontent.com/16364053/155555653-962ac274-191b-40ec-b154-21cdc01fffbd.png)

Reply:
Will use that. Just for my intrest, does it make any other difference?